src/ocean/text/arguments/TimeIntervalArgs.d

```diff
@@ -200,7 +220,7 @@ public TimestampInterval processTimeIntervalArgs ( Arguments args )
 
     return createTimestampInterval(
         args["time-interval"].assigned[0],
-        args["time-interval"].assigned[0],
+        removeTimeDivision(args["time-interval"].assigned[0]),
```

Comment:
Why are you calling `removeTimeDivision` here? Here end = begin = "non relative time"...

Reply:
it's needed because of this case: `--time-interval now/h` which should be the same as `--time-interval now/h now`